src/main/java/uk/ac/ebi/ena/webin/cli/service/RatelimitService.java

```diff
@@ -51,16 +60,25 @@ public RatelimitServiceRequest(String context, String submissionAccountId, Strin
 
     public RateLimitResult ratelimit(String context, String submissionAccountId, String studyId, String sampleId) {
         RestTemplate restTemplate = new RestTemplate();
-        restTemplate.setErrorHandler(new DefaultErrorHander(WebinCliMessage.RATE_LIMIT_SERVICE_SYSTEM_ERROR.text()));
 
```

Comment:
I think we should continue to use setErrorHandler and improve the NotFoundErrorHandler. See previous comment.